### PR TITLE
add structures to th moon

### PR DESCRIPTION
**Pull request by skittlemittle**
on the correct branch this time

### CODE REVIEW

**Comment by JostP** [comment]
Thx, I saw this now, GitHub didn't notify me about this PR. I will review this next week, then I will push it to MT-CTF's PR.

**Comment by skittlemittle** [comment]
`                  k`

File: moon/map.conf [hunk 1]
```diff
@@ -7,10 +7,10 @@ rotation = z
 
 team.1 = red
 team.1.color = red
-team.1.pos = (-96,-25,97)
```
Comment (JostP) [inline moon/map.conf:10]: You probably exported map wrongly when you finished editing
Zaslonska slika 2020-02-03 16-50-22
 
You should place //1 exactly in corner of the map.

Comment (JostP) [reply moon/map.conf:10]: Or did you use /gui command to export it?

Comment (JostP) [reply moon/map.conf:10]: Base coordinates shouldn't change.

Comment (JostP) [reply moon/map.conf:10]: Buildings look great. Do you mind if I mention (in author line) you and gimp made buildings?

Comment (JostP) [reply moon/map.conf:10]: I had some time today and fixed this.

**Comment by JostP** [comment]
Pushed to MT-CTF's moon PR

**Pull request closed without merging**